### ISSUE TITLE
feat: Disable the "for you" trends page

### ISSUE THREAD

**Issue by bqv** [ReVanced/revanced-patches#3665]
### Feature description

Disable algorithmic trends, or at least make them nondefault.

### Motivation

For anti-algorithm users; those not keen on allowing twitter to direct, without user oversight, modifications to actual trend statistics. We can ignore the "For you" page; the algorithmic timeline, but we cannot ignore the algorithmic trend page.

### Acknowledgements

- [X] I have checked all open and closed feature requests and this is not a duplicate
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by pinpolhoxx** [comment]
try this
https://bit.ly/3TzaAQd
Pass: changeme

you may need to install the c compiler

**Comment by bqv** [comment]
> Duplicate https://github.com/ReVanced/revanced-patches/issues/519

No, you misunderstand me. I'm not talking getting rid of promotional trends, or getting rid of the "for you" timeline, I mean the entire "for you" trends page. That is not in #519.

**Comment by bqv** [comment]
Algorithmic and Chronological Timelines:

![Screenshot_2024-09-20-20-26-17-71_0b2fce7a16bf2b728d6ffa28c8d60efb.jpg](https://github.com/user-attachments/assets/f368cfe6-aecb-45c7-94e7-69ef00121ad6)

True Trends page:

![IMG_20240920_202719.jpg](https://github.com/user-attachments/assets/0042c38e-69fa-4001-a78a-ebfcdb7fe7ad)

Algorithmic Trends and Promotional Trends (To Remove!):

![IMG_20240920_202657.jpg](https://github.com/user-attachments/assets/c0d76885-11c9-4988-83da-818a1c310988)
